contracts/v4/OmsPolicy.sol

```diff
@@ -73,6 +88,39 @@ contract OmsPolicy is Ownable {
         _;
     }
 
+    modifier onlyAdmin() {
+        require(admins[msg.sender] == true, "Not admin role");
+        _;
+    }
+
+    // set Admin
+    function setAdmin(address _admin, bool _status)
+        external
+        onlyOwner
+    {
+        require(admins[_admin] != _status, "already admin role");
+        admins[_admin] = _status;
+    }
+
+    // set target price
+    function setTargetPrice(uint256 _targetPrice)
+        external
+        onlyAdmin
+    {
+        lastTargetPrice = TARGET_RATE;
+        TARGET_RATE = _targetPrice;
+        emit LogTargetPrice(lastTargetPrice, TARGET_RATE, block.timestamp);
```

Comment:
Why would you want to change the target rate? Isn't it fixed to 1 USD?

Reply:
The true value of the USD changes over time based on inflation. What we are doing now is adjusting the target price relative to the performance of the USD against other world currencies.

Reply:
I know the next thing on your mind... yes, we have to update the dashboard to fetch the target rate now since it's not fixed to a dollar.